index.html

```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
             <button class="button-1" type="button">Start Collaboration</button>
     </section>
     <img src="img/image_geometry_2.svg" alt="background" class="image">
-</main>  
+    <section class="section-2">
+        <h1 class="heading-1">Projects</h1>
+        <div class="background-color">
+            <img src="img/Rectangle21.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+        <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button" >
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>
+        <div class="background-color">
+            <img src="img/Rectangle22.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button" >
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
```

Comment:
- [ ] The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.